src/controllers/user-credential.controller.ts

```diff
@@ -95,42 +93,26 @@ export class UserCredentialController {
         // Add new credential
         const twitterCredential = await this.createCredential(twitterUser);
 
-        return this.transferTips(twitterCredential);
-
-        // const statusTransfer = await this.transferTipsToUser(twitterCredential, twitterUser.id)
-
-        // if(!statusTransfer) throw new HttpErrors.NotFound('RPC Lost Connection')
-
-        // if (twitterUser.id) this.fetchFollowing(twitterUser.id);
+        this.transferTips(twitterCredential);
 
-        // return true
+        return true
 
       case 'reddit':
```

Comment:
untuk yang hardcode `reddit` mitol tambahin comment `//TODO: move to single constant enum platform`